sdk/storage/azure-storage-blob/pom.xml

```diff
@@ -313,6 +313,7 @@
                 --add-opens com.azure.storage.common/com.azure.storage.common.implementation=ALL-UNNAMED
                 --add-opens com.azure.storage.common/com.azure.storage.common.sas=ALL-UNNAMED
                 --add-opens com.azure.storage.blob/com.azure.storage.blob=ALL-UNNAMED
+                --add-opens com.azure.storage.blob/com.azure.storage.blob=com.fasterxml.jackson.databind
```

Comment:
What was causing the test to fail? Was test code inaccessible to Jackson or source classes? If it was the latter this should be an `opens to` in the `module-info`.

Reply:
@alzimmermsft this line https://github.com/Azure/azure-sdk-for-java/blob/77efff64a75d8c97041e69d587d077451827033a/sdk/storage/azure-storage-blob/src/test/java/com/azure/storage/blob/ImmutableStorageWithVersioningTest.groovy#L90

Reply:
It seems that https://github.com/Azure/azure-sdk-for-java/pull/23125 is also fixing the problem, so we'll try that first and use this PR as a backup.